Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -127,22 +129,35 @@ public class HTTPServer: Server {
 
     var quiescingHelper: ServerQuiescingHelper?
 
+    private var ctx: ChannelHandlerContext?
+    
+    /// server configuration
+    public var serverConfig: HTTPServerConfiguration
+    
+    //counter for no of connections
+    var connectionCount = Atomic(value: 0)
+
+    // The data to be written as a part of the response.
+    //private var buffer: ByteBuffer
+
     /**
      Creates an HTTP server object.
 
      ### Usage Example: ###
      ````swift
-     let server = HTTPServer()
+     let config =HTTPServerConfiguration(requestSize: 1000, coonectionLimit: 100)
+     let server = HTTPServer(serverconfig: config)
      server.listen(on: 8080)
      ````
     */
-    public init() {
+    public init(serverConfig: HTTPServerConfiguration = .default) {
```

Comment:
Kitura-net's initializer does not take a parameter.  Instead, `HTTPServer.options` has an initial value of the default options, and is set by the user after the server is created, if desired.
(See usage in https://github.com/IBM-Swift/Kitura/pull/1481)